apps/developer-hub/src/components/HomepageHero/HeroGraphic/index.module.scss

```diff
@@ -0,0 +1,30 @@
+@use "@pythnetwork/component-library/theme";
+
+.heroGraphic {
+  position: relative;
+  width: 100%;
+  height: 400px;
```

Comment:
I recommend avoiding using px values; we should ideally always stick to using sizes via theme.spacing but at the very least, unless we're styling something that must be sized in px, we should be using rem